open_instruct/grpo_fast.py

```diff
@@ -2122,13 +2115,21 @@ def main(args: Args, tc: TokenizerConfig, model_config: ModelConfig, num_eval_sa
     reward_fn = make_reward_fn(args)
 
     # Start vLLM engines to process from queues
+    if args.num_unique_prompts_rollout * args.num_samples_per_prompt_rollout % args.vllm_num_engines != 0:
```

Comment:
nit: could we add this check to the post_init so this fires as soon as the user runs the code, instead of partway through setup?

Reply:
yes! Can you point me where I do that?

Reply:
here? https://github.com/allenai/open-instruct/blob/main/open_instruct/grpo_fast.py#L402